sdk/communication/communication-verified-recipients/review/communication-verified-recipients.api.md

```diff
@@ -0,0 +1,112 @@
+## API Report File for "@azure-tools/communication-verified-recipients"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference lib="esnext.asynciterable" />
+
+import { CommonClientOptions } from '@azure/core-client';
+import * as coreClient from '@azure/core-client';
+import { KeyCredential } from '@azure/core-auth';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export interface AcsVerification {
+    attempts?: number;
+    challengeId?: string;
+    identity?: string;
+    immutableId?: string;
+    retries?: number;
+    status?: string;
+    subscriptionId?: string;
+    verificationId?: string;
+}
+
+// @public
+export interface AcsVerificationDeleteVerificationOptionalParams extends coreClient.OperationOptions {
+}
+
+// @public
+export interface AcsVerificationGetVerificationConstantsOptionalParams extends coreClient.OperationOptions {
+}
+
+// @public
+export type AcsVerificationGetVerificationConstantsResponse = VerificationConstantsResponse;
+
+// @public
+export interface AcsVerificationGetVerificationsOptionalParams extends coreClient.OperationOptions {
+}
+
+// @public
+export type AcsVerificationGetVerificationsResponse = AcsVerification[];
+
+// @public (undocumented)
+export const AcsVerificationMapper: coreClient.CompositeMapper;
+
+// @public
+export interface AcsVerificationRequestVerificationOptionalParams extends coreClient.OperationOptions {
+    channel?: string;
+    identity?: string;
+}
+
+// @public
+export type AcsVerificationRequestVerificationResponse = VerificationResponse[];
+
+// @public
+export interface AcsVerificationVerifyIdentityOptionalParams extends coreClient.OperationOptions {
+    verificationCode?: string;
+}
+
+// @public
+export type AcsVerificationVerifyIdentityResponse = VerificationResponse;
+
+// @public (undocumented)
+export const ConstantsMapper: coreClient.CompositeMapper;
+
+// @public (undocumented)
+export const VerificationCodeRequestMapper: coreClient.CompositeMapper;
+
+// @public
+export interface VerificationConstantsResponse {
+    currentNumberOfVerifications?: number;
+    maxRetriesAllowed?: number;
+    maxVerificationsAllowed?: number;
+}
+
+// @public (undocumented)
+export const VerificationRequestMapper: coreClient.CompositeMapper;
+
+// @public
+export interface VerificationResponse {
+    isVerified?: boolean;
+    verificationId?: string;
+}
+
+// @public (undocumented)
+export const VerificationResponseMapper: coreClient.CompositeMapper;
+
+// @public (undocumented)
```

Comment:
Could you add docs for the undocumented parts?

Reply:
what do you mean here? @HarshaNalluru can you explain further

Reply:
oh ok cool